scripts/aws.hs

```diff
@@ -105,9 +125,10 @@ runS3 CheckDaedalusReleaseURLs bucket = do
   printf ("  OS X:   "%s%"\n") (ppURL $ osxLive bucket)
   printf ("  Win64:  "%s%"\n") (ppURL $ w64Live bucket)
 
-runS3 (SetDaedalusReleaseBuild rsOSX@(OSX id) rsWin64@(Win64 avPath)) bucket = do
-  let (osxurl@(URL osxpr osxcn osxpath), w64url@(URL w64pr w64cn w64path)) =
-        (buildURL id, avpathURL avPath)
+runS3 (SetDaedalusReleaseBuild debug'mode rsOSX@(R_OSX osx'bid) rsWin64@(R_Win64 w64'bid avId)) bucket = do
+  let (   osxurl@(URL osxpr osxcn osxpath)
```

Comment:
Can we name these variables to represent their values? It's really hard to follow the code with `osxpr` and `osxcn`, etc